### PR TITLE
feat: compress chunk n+1 like chunk n

### PR DESCRIPTION
**Pull request by danking**
EDIT: still needs some work, sorry for the ping

### CODE REVIEW

File: vortex-sampling-compressor/src/compressors/chunked.rs [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+impl EncodingCompressor for ChunkedCompressor {
+    fn id(&self) -> &str {
+        Chunked::ID.as_ref()
+    }
+
+    fn cost(&self) -> u8 {
+        0
+    }
+
+    fn can_compress(&self, _array: &Array) -> Option<&dyn EncodingCompressor> {
```
Comment (robert3005) [inline vortex-sampling-compressor/src/compressors/chunked.rs:69]: you could say here that you can only compress chunked arrays and remove the filter in the main compressor function

Comment (danking) [reply vortex-sampling-compressor/src/compressors/chunked.rs:69]: If I do that, I'll never be called because the sample is never a chunked array :|

Comment (robert3005) [reply vortex-sampling-compressor/src/compressors/chunked.rs:69]: oof, forgot that sample is contiguouos

**Comment by danking** [comment]
A summary of the vortex compression situation as of 52800a2

| Benchmark suite | vortex:zstd PR | vortex:zstd PR:develop | time PR:develop
|-|-|-|-|
| `taxi` | `1.02` |  `1.08` | `0.78` |
| `AirlineSentiment` | `5.20` |  `0.78` | `0.95` |
| `Arade` | `0.49` |  `1.03` | `0.82` |
| `Bimbo` | `1.38` |  `1.16` | `0.89` |
| `CMSprovider` | `1.19` |  `1.08` | `0.86` |
| `Euro2016` | `1.42` |  `1.03` | `1.06` |
| `Food` | `1.37` | `1.10` | `0.89` |
| `HashTags` | `1.67` |  `1.10` | `0.93` |
| `l_comment` | `1.35` | `1.16` | `0.86` |

**Comment by robert3005** [comment]
I don't quite follow the table you've made, are the first two columns compression ratios vs what and the last one is time vs?

**Comment by danking** [comment]
1. First column is, in this PR, what's the ratio of Vortex (compressed) size to Parquet-with-zstd size.
2. Second column is a ratio of ratios of sizes: `(vortex-PR/parquet_zstd) / (vortex-develop/parquet_zstd)`
3. Third column is time to run Vortex compress in this PR divided by that time using origin/develop.

**Pull request closed without merging**